src/lib/collectors/utils/requester.ts

```diff
@@ -41,11 +42,14 @@ export class Requester {
     private _request: request;
     /** Internal `redirectManager`. */
     private _redirects = new RedirectManager();
+    /** Maximum number of redirects */
+    private _limit;
 
-    constructor(customOptions = {}) {
+    constructor(customOptions = {}, redirectsLimit = 3) {
```

Comment:
I think `redirectsLimit` should be part of `customOptions` and be named `maxRedirects` for consistency with the `request` module. Also the default value should be 10 to match `request` again.